crowbar_framework/app/controllers/network_controller.rb

```diff
@@ -16,6 +16,8 @@
 #
 
 class NetworkController < BarclampController
+  skip_before_filter :enforce_installer
```

Comment:
i will try it again without. i have added it because install-chef-suse failed at the admin ip check, and this fixed it somehow